### ISSUE TITLE
Bootstrap affects Vis Network styling?

### ISSUE THREAD

**Issue by FilipQL** [almende/vis#2640]
I am using Vis Network and although the Bootstrap CSS file (bootstrap.min.css) is included first (before vis.min.css):
```
<link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/vis/4.18.0/vis.min.css">
```
... the graph (**toolbar**) is affected:

![1](https://cloud.githubusercontent.com/assets/18264428/22355555/8e1f4aa6-e42b-11e6-8cff-21f58d6ea2e8.png)

![2](https://cloud.githubusercontent.com/assets/18264428/22355560/973c7690-e42b-11e6-88a6-c1824e006fd9.png)

Here is the demo: https://jsfiddle.net/Filip_Z/wef6L1c8/

Do you know how can I prevent this?

**Comment by mojoaxel** [comment]
I can confirm this. I'll have a look.

**Comment by mojoaxel** [comment]
This is coursed because bootstrap sets all DOM elements to `box-sizing: border-box`. I'll provide a fix.

**Comment by mojoaxel** [comment]
provides fix #2654
